tests/bdd/features/volume/resize/resize_offline.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Volume resize
+
+    Background:
+        Given a deployer cluster
+
+    Scenario Outline: expand an unpublished volume
+        Given an unpublished volume with <repl_count> replicas and all are healthy
+        When the volume is expanded
+        Then all the replicas of volume should be resized to new capacity
+        And the volume expand status should be success
+        And the volume is expanded to the new capacity
+        Examples:
+        | repl_count |
+        | 1          |
+        | 3          |
+
+
+    Scenario: expand a published volume
+        Given a published volume with more than one replica and all are healthy
+        When the volume is expanded
+        Then the volume expand status should be failure
+        And the failure reason should be volume-in-use precondition
+
+
+    Scenario: expand an unpublished volume and then make it published
+        Given an unpublished volume with more than one replica and all are healthy
+        When the volume is expanded
+        Then all the replicas of volume should be resized to new capacity
```

Comment:
You can reuse the exact same wording as from scenario 1, this way you can reduce code!